### ISSUE TITLE
contra-hdsl shared library

### ISSUE THREAD

**Issue by lioramilbaum** [CentOS-PaaS-SIG/contra-env-setup#122]
The automation should support a DEV environment with contra-hdsl configured as jenkins shared libraries. @ifireball

**Comment by arilivigni** [comment]
We already setup the helper containers that supports contra-hdsl.

**Comment by lioramilbaum** [comment]
What are the helper containers? 
Do we have an automation which set a DEV environment following the instructions listed [here](https://github.com/openshift/contra-hdsl/blob/master/README.md)?

**Comment by arilivigni** [comment]
Then that would be a config file provided with the contra-hdsl, but would
not live in this repo.

-== @ri ==-

**Comment by lioramilbaum** [comment]
Are you saying there is no full automation which spawns a DEV environment? I am asking it again because I don't understand your answer.

**Comment by arilivigni** [comment]
Actually now thinking about it the only step missing is loading the contra-hdsl library globally.  Besides that the helper containers linchpin-executor and ansible-executor are loaded.

**Comment by lioramilbaum** [comment]
So the helper (HDSL) containers are linchpin-executor and ansible-executor. Starting to follow.
I will dig in to implement the step which is missing while removing the steps which loads the unneeded libraries.
